lib/index.js.flow

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
Sample generated `.js.flow` file that would ideally be shipped with our packages and allow consumers to import our types.